lib/utils.coffee

```diff
@@ -14,17 +26,30 @@ isHaskellSource = (fname) ->
     return true
   return false
 
+getElementsByClass = (elem,klass) ->
+  # this is pretty obtuse, but jquery doesn't seem to support inspecting the shadowRoot directly
+  # can remove the fallback once the option to disable the shadow dom is gone
```

Comment:
You might want to drop jQuery altogether and use `rootElement` instead of `shadowRoot`, which will get you root element for shadow dom or otherwise. You might also consider using `querySelectorAll` instead of `getElementsByClassName`, which has semantics almost exactly like `jQuery.find`